src/content/ai-agent-fleets/ai-agent-fleets-admin-config.mdx

```diff
@@ -0,0 +1,238 @@
+- An API key from either Anthropic or AWS (for Amazon Bedrock)
+- A Git token for repository integration (if not, we'll help you set one up)
+
+## Enabling AI Agent Fleets for your organization
+
+### Step 1: Enable AI Agent Fleets
+
+- From the Okteto Admin Dashboard, navigate to **Admin -> AI Agent Fleets** underneath the Settings Section
+- Toggle the AI Agent Fleets switch to enable the feature for your organization
+
+<p align="center">
+  <Image
+    src={require("@site/static/img/admin_ai_agent_fleets.png").default}
+    alt="AI Agent Fleets Admin Dashboard"
+    width="800"
+  />
+</p>
+
+
+### Step 2: Configure Access Control
+
+Choose who can use AI Agent Fleets in your organization:
+
+#### Option A: Enable for All Users
+
+- Select All users to grant access to everyone in your organization
+- This is recommended for smaller teams or companies already familiar with the product
+
+#### Option B: Enable for Selected Users
+
+- Select `Selected users`
+- Click the users view link to navigate to the user management tab
+- Select the specific users who should have access to AI Agent Fleets
+
+Note: Users without access will not see the Agents tab in their dashboard.
+
+### Step 4: Configure Your LLM Provider
+
+AI Agent Fleets requires an LLM provider to power the Okteto AI Agents. Choose between Anthropic (direct) or via Amazon Bedrock.
+
+#### Option A: Anthropic Configuration
+Use this option for direct integration with Anthropic's API.
+
+<p align="center">
+  <Image
+    src={require("@site/static/img/admin_ai_llm_provider_config_anthropic.png").default}
+    alt="AI Agent Fleets LLM Provider Configuration"
+    width="800"
+  />
+</p>
+
+1. **Select Provider**: Choose **Anthropic** from the Provider dropdown
+2. Obtain API Key:
+  - Click the "Get your Anthropic API key →" link
+  - Sign in to your [Anthropic account](https://console.anthropic.com/) (or create one)
+  - Navigate to [API Keys section](https://console.anthropic.com/settings/keys)
+  - Create a new API key with appropriate permissions
+  - Copy the key
+
+3. **Enter API Key**: Click Update and paste your Anthropic API key in the field
+4. **Save Configuration**: Click Save to save your settings
+
+#### Option B: Amazon Bedrock Configuration
+Use this option if your organization prefers AWS-managed AI services.
+
+<p align="center">
+  <Image
+    src={require("@site/static/img/admin_ai_llm_provider_config_bedrock.png").default}
+    alt="AI Agent Fleets LLM Provider Configuration for Bedrock"
+    width="800"
+  />
+</p>
+
+1. **Select Provider**: Choose **Amazon Bedrock** from the Provider dropdown
+2. **Configure Region**:
+  - Select the AWS region where you have Bedrock enabled
+  - Common regions: `us-east-1`, `us-west-2`, `eu-west-1`
+  - Ensure Claude models are available in your selected region
+
+3. **Set Up AWS Credentials**:
+  - Click "Get your Bedrock API key →" for setup instructions
+  - You'll need either:
+    - AWS Access Key ID and Secret Access Key
```

Comment:
Bedrock doesn't work this way. When users follows the AWS ink, they end up having a BedRock API Key, and that's the only thing they need to paste in the Okteto Admin view in the API Key field